WORKSPACE

```diff
@@ -37,12 +37,14 @@ bind(
     actual = "//external:ssl",
 )
 
-ENVOY_SHA = "b02d8a9b5033a61a7b13d98999f5f00c4b85b6a4"  # Sep 26, 2017 (use github to download tclap instead of sourceforge)
 
-http_archive(
+ENVOY_SHA = "577410987eddf1d8f47439475c8569dcd68eeb04"  # Cherry-pick Oct 20
+# Branched from "b02d8a9b5033a61a7b13d98999f5f00c4b85b6a4"  # Sep 26, 2017 (use github to download tclap instead of sourceforge)
+
+git_repository(
```

Comment:
Use `http_archive`, this is an intended workaround for Cloud Container Builder.

Reply:
This is using istio/envoy fork - is the archive auto-generated ? And what is the issue with GCB ?

Reply:
Yes same URL format works.

https://github.com/istio/envoy/archive/577410987eddf1d8f47439475c8569dcd68eeb04.zip

The root cause: https://github.com/bazelbuild/bazel/issues/3786